### DISCUSSION TITLE
Azure無料期間を過ぎて課金されないようにする方法を知りたい

### DISCUSSION THREAD

**Discussion by kazujoy** [dzeyelid/onsen-maas-vueapp#18] [Q&A] [answered]
今回、Azureアカウントを初めて作成したのですが、無料期間を過ぎて課金されないようにするにはどうしておいたらよいでしょうか？

**Comment by uikou** [comment, marked as answer]
無償版を作成された場合は、止まるだけなので課金されませんのでご安心ください。（カードなどを求められても、本人確認用なので）
会社のサブスクリプションに紐づけてアカウントなどを利用されている場合は、サービスを止めていただくことをおすすめします。

**Reply by kazujoy** [reply]
ああ、確かに。そうなんですね。ありがとうございます！

**Comment by uikou** [comment]
補足です。今回 Static Web Apps はフリープランがあり、今回はそれを使っているのでそれ以外のところは今回のハンズオンで気になる場合は最後のクリーンアップを実施してください。
以下クリーンアップのリファレンスです↓
https://github.com/dzeyelid/onsen-maas-vueapp/blob/main/docs/self-paced-handson/cleanup-resources.md
